### PR TITLE
Voltage testing

### PR DESCRIPTION
**Pull request by manthanand**
# Quality Assurance Checklist 
To make reviews more efficient, please make sure the software feature meets the following standards and check everything off that meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._

There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 

## Requirements 
- [x] Followed Coding Style Guide 
- [x] Code Build checks pass 
- [x] No merge conflicts 
- [x] Software feature has test associated with it. 
- [x] Software feature has documentation for it updated.
- [x] If applicable, does software feature have simulation test associated with it (.json & out.json file)
- [x] Test provides useful information and uses relevant data to accurately represent BPS 
    - NOTE: If test file already exists, use that one 
- [x] 2 Members of the team have already reviewed my PR 
    - NOTE: Approver will not look at PR if 2 other members haven't reviewed first 
- [x] Did you Have Fun?

## Things to Consider 
- [x] Even if the above are checked, is this the best way of writing my code? 
    - It's possible to write code that works, but are there ways to make code more efficient?

### CODE REVIEW

**Review by MichaelMohn — DISMISSED** [review]
looks good to me

**Review by Champers5000 — APPROVED** [review]
I don't see any problems if it runs correctly

File: Apps/Src/Temperature.c [hunk 1]
```diff
@@ -390,17 +389,15 @@ int32_t Temperature_GetTotalPackAvgTemperature(void){
  * @return SUCCESS or ERROR
  */
 ErrorStatus Temperature_SampleADC(uint8_t ADCMode) {
-#ifndef SIMULATION
-    //take control of mutex
-      RTOS_BPS_MutexPend(&MinionsASIC_Mutex, OS_OPT_PEND_BLOCKING);
-    
-    LTC6811_adax(ADCMode, AUX_CH_GPIO1);							// Start ADC conversion on GPIO1
-    LTC6811_pollAdc();
```
Comment (ray4477) [inline Apps/Src/Temperature.c:398]: why is LTC6811_pollAdc() removed? Im actually kinda curious

File: Makefile [hunk 2]
```diff
@@ -56,7 +56,7 @@ help:
 	@echo -e "When building for the board, you can specify custom define values for config.h."
 	@echo -e "See Config/Inc/config.h for all values."
 	@echo -e "Specify with the following ${PURPLE}format${NC}(quotes ARE needed):"
-	@echo -e "		${ORANGE}make ${BLUE}DEFINES=${GREEN}\"-D VARIABLE=VALUE VARIABLE=VALUE\"\n"
+	@echo -e "		${ORANGE}make ${BLUE}DEFINES=${GREEN}\"-D VARIABLE=VALUE -D VARIABLE=VALUE\"\n"
```
Comment (connorl309) [inline Makefile:59]: Once this gets pushed I can update the help print in my makefile branch, its DEFINES=<name>=VALUE/<name>=VALUE etc

**Review by connorl309 — APPROVED** [review]
lgtm whitespace adjustments are nice. no actual functionality changes i have Qs about

File: Apps/Src/Voltage.c [hunk 3]
```diff
@@ -53,19 +53,19 @@ void Voltage_Init(cell_asic *boards){
...
     // Write Configuration Register
     LTC6811_wrcfg(NUM_MINIONS, Minions);
     //release mutex
-      RTOS_BPS_MutexPost(&MinionsASIC_Mutex, OS_OPT_POST_NONE);
+    RTOS_BPS_MutexPost(&MinionsASIC_Mutex, OS_OPT_POST_NONE);
 
-    // Read Configuration Register
-    // take control of mutex
-      RTOS_BPS_MutexPend(&MinionsASIC_Mutex, OS_OPT_PEND_BLOCKING);
     wakeup_sleep(NUM_MINIONS);
```
Comment (ray4477) [inline Apps/Src/Voltage.c:62]: do we need this wakeup_sleep() ? its already called in this function at the beginning

File: BSP/STM32F413/Src/BSP_WDTimer.c [hunk 4]
```diff
@@ -13,7 +13,7 @@ void BSP_WDTimer_Init(void) {
     // IWDG has a timeout value of 4.096 / 8 seconds (about 500 ms).
 	IWDG_WriteAccessCmd(IWDG_WriteAccess_Enable);       // Enable access to the IWDG_PR and IWDG_RLR registers
 	IWDG_SetPrescaler(IWDG_Prescaler_32);               // Prescaler divider feeding the counter clock
-	IWDG_SetReload(IWDG_RLR_RL / 8);                    // Reload value set to 0xFFF / 8
+	IWDG_SetReload(IWDG_RLR_RL / 4);                    // Reload value set to 0xFFF / 8
```
Comment (manthanand) [inline BSP/STM32F413/Src/BSP_WDTimer.c:16]: made watchdog slower

Comment (tasniabari) [inline BSP/STM32F413/Src/BSP_WDTimer.c:16]: I think I aked you, but just a reminder to comment why this change was made in the timer :)

**Review by ray4477 — APPROVED** [review]
I think it's good, other than the comments I left below such as the wakeup_sleep, (mostly probably just needs clarification). Mostly just indentation and datatype changes.

**Review by tasniabari — APPROVED** [review]
Looks good

File: Apps/Src/Temperature.c (hunk 1 above)
Comment (manthanand) [reply Apps/Src/Temperature.c:398]: good catch

File: Apps/Src/Voltage.c (hunk 3 above)
Comment (manthanand) [reply Apps/Src/Voltage.c:62]: if a thread interrupts in that time, then we will wait 10ms and the ltc6811 will have gone back to sleep
